web/profiles/joinup/config/install/search_api.server.collections_search_server.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 9fe2d8ff-5921-4ec5-a400-c5351a1499f4
+langcode: en
+status: true
+dependencies:
+  module:
+    - search_api_db
+id: collections_search_server
+name: 'Collections Search Server'
```

Comment:
Multiple servers can be created through UI and each having a separate index or indexes. So We can have multiple instances of local database servers or have a central one with multiple Indexes. It is all a matter of how we define it. What do you think?